### PR TITLE
phone size responsive issue shifting details page #8831 Open

### LINKED ISSUE #8831: phone size responsive issue shifting details page
**Describe the bug**
responsive issue on shifting details page, 

**To Reproduce**

1. Go to 'shifting on sidebar'
2. Click on 'all details button from any card and head to shift details page'
                  'in phone size responsive issue'

**Expected behavior**
clear and better mobile user experience

**Screenshots**
Image

**Desktop (please complete the following information):**

- OS: [windows]
- Browser [chrome]
- Version [129.0.6668.101]

**Additional context**

### PR DESCRIPTION
**Pull request by ayushpatil2122**
## Proposed Changes

- Fixes #8831
- Change 1
Added a Tailwind css classes 

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Summary by CodeRabbit

- **New Features**
	- Enhanced layout and styling of the Shift Details component for improved user experience.
	- Updated buttons with responsive width classes for better adaptability on various screen sizes.

- **Bug Fixes**
	- Streamlined conditions for disabling the update button while retaining tooltip functionality.

- **Chores**
	- Updated the `husky` package version for routine maintenance.

### CODE REVIEW

File: src/components/Shifting/ShiftDetails.tsx [hunk 1]
```diff
@@ -478,30 +478,24 @@ export default function ShiftDetails(props: { id: string }) {
       ) : (
         <Page
           title={t("shifting_details")}
-          crumbsReplacements={{
-            [props.id]: { name: data?.patient_object?.name },
-          }}
```
Comment (rithviknishad) [inline src/components/Shifting/ShiftDetails.tsx]: why were these deleted?

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -131,7 +131,7 @@
     "eslint-plugin-react": "^7.37.2",
     "eslint-plugin-react-hooks": "^5.0.0",
     "glob": "^11.0.0",
-    "husky": "^9.1.6",
+    "husky": "^9.1.7",
```
Comment (rithviknishad) [inline package.json:134]: was this intentionally upgraded?

Comment (ayushpatil2122) [reply package.json:134]: this given me error when i was pushing this code

Comment (ayushpatil2122) [reply package.json:134]: so i installed it again

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**